reframe/frontend/printer.py

```diff
@@ -24,6 +24,10 @@ def __init__(self):
         self.line_width = 78
         self.status_width = 10
 
+    def init_progress(self, num_cases):
+        self._progress_count = 0
+        self._total_count = num_cases
```

Comment:
Just some naming suggestions:

```suggestion
    def reset_progress(self, total_cases):
        self._progress_count = 0
        self._progress_total = total_cases
```